django/src/rdwatch/views/model_run.py

```diff
@@ -456,3 +515,32 @@ def get_modelrun_evaluations(request: HttpRequest, hyper_parameters_id: int):
         }
     query['region'] = model_run['region']
     return 200, query
+
+
+@router.post('/{id}/download')
```

Comment:
The current convention elsewhere in the codebase is to use trailing slashes (applies to the other endpoints introduced in this PR too)
```suggestion
@router.post('/{id}/download/')
```